### DISCUSSION TITLE
Best guess for Coil 2.0 release

### DISCUSSION THREAD

**Discussion by yschimke** [coil-kt/coil#1097] [General]
Just curious if you have a best guess for Coil 2.0 availability. I'm using 1.4 but as I'm digging into the caching I'm wondering whether 2.0 is more useful.  If you have a sense, I'd be basically interested whether it is...

- Imminent - in which can I should switch
- Way off - in which case I should get things working on 1.4 for now.

Also happy with unknown as the answer.

**Comment by colinrtwhite** [comment]
2.0 has a ton of improvements over 1.x so I'd highly recommend switching if you're OK with some breaking API changes (largely with the Compose APIs) in the next few weeks. Coil 2.x is stable overall and we've been using it in production at Instacart for a while.

I'd say the 2.x release is ASAP (likely in the next 30 days). I'm hoping there'll be 1 more alpha then a release candidate. Originally I wanted to release before the end of 2021, but got some feedback from folks in ASG with respect to the Compose APIs. The core artifact is good to go, but I've been getting feedback on https://github.com/coil-kt/coil/pull/1048 for the last month which has delayed the release slightly.
